### ISSUE TITLE
Add a map category

### ISSUE THREAD

**Issue by yohanboniface** [umap-project/umap#2283]
The idea is to allow users to define one or more categories for their map. This may help discovering useful maps, eg. by allowing to filter the search by category, or by creating some pages like "last maps in category x" or "most starred maps in category x".

In my idea, the category list is closed, defined by the uMap instance administrator, and I'm in favor of using a setting, so it's possible to use the translation on the labels (if we use a DB model for instance, we could not use the current transifex workflow).

Here is a first suggestion for the default list (please complete):
- Art and Culture
- Bike
- Environment
- Education
- Public sector
- Sport and leisure
- Travel
- Trekking
- Tourism

**Comment by clwacker** [comment]
Very good idea. One more suggestion for the list:
- Cultural Heritage & History

**Comment by yohanboniface** [comment]
Maybe something like "Food" ? I see a lot of maps with local food and farms, and so on.

**Comment by differentieel** [comment]
maybe leave te name of a category to the user to choose how and in which language the user fits the best

**Comment by Aurelie-Jallut** [comment]
We could present them like this, so that you can see the filters straight away.

![Image](https://github.com/user-attachments/assets/17792d4e-78a6-46a2-a53d-0ed3ca4d275c)

**Comment by yohanboniface** [comment]
@Aurelie-Jallut a few questions/remarks:

- I wonder with this mockup how convenient it will be to access to the left most tags ? I'm not sure how many tags we'll have, how long their labels will be, and each instance may configure their own (see this discussion for the ANCT instance: https://github.com/umap-project/umap-aic/issues/20 )
- should the filtering be also available in the dashboard page ? So one can filter their own maps, and in this case we may want a consistent system to filter
- should we display the map filters in the "about" panel ? And if yes, should they be clickable and link to a "filter" page ?

**Comment by yohanboniface** [comment]
@Aurelie-Jallut we are experimenting with adding icons for tags, wdyt ?

![Image](https://github.com/user-attachments/assets/364e9a61-253f-4270-b6b1-09d2e106205b)

Also, what about displaying tags in map lists / search results ?

**Comment by Aurelie-Jallut** [comment]
> 1- I wonder with this mockup how convenient it will be to access to the left most tags ? I'm not sure how many tags we'll have, how long their labels will be, and each instance may configure their own (see this discussion for the ANCT instance: https://github.com/umap-project/umap-aic/issues/20 )

Good point, I just prefer to show things when possible, for example they could be displayed in the left-hand area. 

> 2- should the filtering be also available in the dashboard page ? So one can filter their own maps, and in this case we may want a consistent system to filter

If we want to have the same filters component (the reverse wouldn't be inconsistent for the user, as we have different pages (list and table) and different uses (viewing, editing)), but if we want to have the same filters component, would we also want to allow other columns to be filtered in the dashboard? for example ‘Owner’?

> 3- Should we display the map filters in the ‘About’ panel? And if so, should they be clickable and link to a ‘filter’ page?

Yes, I think that would be interesting, and I wonder if we shouldn't open a link in a new tab in this case (and do the same for the link to the owner cards) ?

> 4- we're experimenting with adding icons for tags, aren't we? 

yes great in your screenshot ! i wonder if it might be difficult to find the right tags for all the themes? 

5- Also, what about displaying tags in map lists and search results?
=> let's mock up this !
